MoMMI/Modules/CodeHandling/dm.py

```diff
@@ -68,6 +69,10 @@ def __init__(self) -> None:
                     continue
 
                 lines[index] = "\t" + line
+                if line.find('#include "'):
```

Comment:
I would recommend using regex and `finditer` so you can make the parsing easier and allow multiple files.